app/_src/kubernetes-ingress-controller/upgrade/kic.md

```diff
@@ -73,6 +73,33 @@ complete the following steps to upgrade to version v1.1 of the CRD:
 
 {% endif_version %}
 
+{% if_version gte:3.4.x %}
+
+### Update Gateway API from v1.1 to v1.2
```

Comment:
Can we reorganize this into nested sections? So that we have 1 `### Upgrade Gateway API` section and then 2 below for each upgrade path

- 1.0 -> 1.1
- 1.1 -> 1.2

WDYT?

Reply:
reorganized the part.